src/widgets/notebooknodeexplorer.cpp

```diff
@@ -2182,6 +2182,29 @@ void NotebookNodeExplorer::setupShortcuts()
         }
     }
 
+    // Copy
+    {
+        auto shortcut = WidgetUtils::createShortcut(coreConfig.getShortcut(CoreConfig::Copy), this);
+        if (shortcut) {
+            connect(shortcut, &QShortcut::activated,
+                    this, [this]() {
+                        copySelectedNodes(false, false);
```

Comment:
Get the second parameter like line 2176~2179.

It means whether it is from the master explorer or the slaver explorer when in separate view mode.

Reply:
done